sim/paladin/item_sets_pve.go

```diff
@@ -187,3 +187,90 @@ var ItemSetMercifulJudgement = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+var ItemSetRadiantJudgement = core.NewItemSet(core.ItemSet{
+	Name: "Radiant Judgement",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			// 2 pieces: Increases damage done by your damaging Judgements by 20% and your Judgements no longer consume your Seals on the target.
+			paladin := agent.(PaladinAgent).GetPaladin()
+			core.MakePermanent(paladin.RegisterAura(core.Aura{
+				Label: "S03 - Item - T2 - Paladin - Retribution 2P Bonus",
+				OnInit: func(aura *core.Aura, sim *core.Simulation) {
+					for _, judgeSpells := range paladin.allJudgeSpells {
+						for _, judgeRankSpell := range judgeSpells {
+							judgeRankSpell.DamageMultiplier *= 1.2
+						}
+					}
+
+					paladin.consumeSealsOnJudge = false
+				},
+			}))
+		},
+		4: func(agent core.Agent) {
+			// 4 pieces: The cooldown on your Judgement is instantly reset if used on a different Seal than your last Judgement.
+			// Implemented in Paladin.go
+			paladin := agent.(PaladinAgent).GetPaladin()
+			core.MakePermanent(paladin.RegisterAura(core.Aura{
+				Label: "S03 - Item - T2 - Paladin - Retribution 4P Bonus",
+				OnInit: func(aura *core.Aura, sim *core.Simulation) {
+
+					originalApplyEffects := paladin.judgement.ApplyEffects
+
+					// Wrap the apply Judgement ApplyEffects with more Effects
+					paladin.judgement.ApplyEffects = func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+
+						sealsLastJudgement := paladin.lastJudgement // Get Last Judgement Seals before it gets overwritten in originalApplyEffects
+						originalApplyEffects(sim, target, spell)
+						sealsThisJudgement := paladin.lastJudgement // Get Active Seals this Judgement (after lastJudgement is set)
+
+						// Two of the possible implementations - TODO figure out actual implementation
+						// anySealsDifferent := sealsThisJudgement != sealsLastJudgement // Would be nice!
+						// allSealsDifferent := (int(sealsThisJudgement) & int(sealsLastJudgement)) == 0 // More conservative option
+						anySealsGained := (int(sealsThisJudgement) & ^int(sealsLastJudgement)) >= 0 // More conservative option (most likely option)
+
+						if anySealsGained {
+							// 4 pieces: The cooldown on your Judgement is instantly reset if used on a different Seal than your last Judgement.
```

Comment:
This comment looks like a copy of the one above. Would you mind removing it?

Reply:
removing